assets/js/components.js

```diff
@@ -47,6 +47,27 @@ jQuery(function () {
         $(td).html(`${rowData.attributes.github_enforce_admins_enabled}`)
       },
     },
+    {
+      data: 'attributes.environments',
+      createdCell: function (td, _cellData, rowData) {
+        const environments = rowData.attributes.environments
+        const prodEnvironments = environments ? environments.filter(env => env.name === 'prod') : []
+
+        let prodSlackChannel = ''
+        if (prodEnvironments.length === 0) {
+          prodSlackChannel = `No Prod environment`
```

Comment:
Don't need backticks as there is no dynamic data in the string

Reply:
Removed backticks